roadmaps/configuration.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: contribute
+title: Bazel Configurability Roadmap
+---
+<style>
+  .padbottom { padding-bottom: 10px; }
+  .etabox {
+    background: #EFEFEF;
+    color: #38761D;
+    font-size: 15px;
+    font-weight: bold;
+    display: inline;
+    padding: 6px;
+    margin-right: 10px;
+  }
+</style>
+
+# Bazel Configurability 2018 Roadmap
+*Last verified: Wednesday, June 6, 2018* ([update history]
+(https://github.com/bazelbuild/bazel-website/commits/master/roadmaps/configuration.md))
+
+This page describes committed plans to advance Bazel's configuration and
+multiplatform support in 2018.
+
+Dates are approximate based on our best understanding of problem complexity
+and developer availability. ETAs will change, but we'll keep them refreshed and
+current.
+
+Have a bug or feature request? File a [GitHub issue
+](https://github.com/bazelbuild/bazel/issues/new).
+
+Questions or suggestions? Email
+<a href="mailto:bazel-discuss@googlegroups.com">bazel-discuss@googlegroups.com</a>.
+
+
+* [Goal](#goal)
+* [Roadmap](#roadmap)
+  * [Platforms](#platforms)
+  * [User-Defined Configuration](#user-defined-configuration)
+  * [Correctness and Speed](#correctness_and_speed)
+
+
+<a name="goal"></a>
+## Goal 
+
+Configurability's goal is to make Bazel a graceful multiplatform build
+tool. It also focuses on how users decide how their projects are built.
+
+
+This translates into the following high-level goals:
+
+1. **`$ bazel {build|test} :all` just works**
+    1. All targets "know" how to build for the right platforms with the right
+       toolchains and desired options
+    1. <div class="padbottom">Users only have to fiddle with options they care
```

Comment:
What is padbottom class doing here?

Reply:
I did it so the final sublist entries aren't closer to the next entry than their own, which I thought was reader-unfriendly:

Without padding:
<img width="695" alt="screen shot 2018-06-07 at 6 17 14 am" src="https://user-images.githubusercontent.com/5130366/41093941-98b69034-6a1a-11e8-8043-3a093d92e1a2.png">

With padding:
<img width="680" alt="screen shot 2018-06-07 at 6 18 58 am" src="https://user-images.githubusercontent.com/5130366/41093989-b6c42578-6a1a-11e8-9e09-aaaae277aa7f.png">

I guess it's not a huge thing, so I'm happy to go with whatever the review crowd prefers.

Reply:
This is fine with me. Thanks for the comparative screenshots.